tests/integration/helpers.py

```diff
@@ -144,3 +165,46 @@ def oci_image(metadata_file: str, image_name: str) -> str:
         raise ValueError("Upstream source not found")
 
     return upstream_source
+
+
+def initial_workload_is_ready(ops_test, app_names) -> bool:
+    """Checks that the initial workload (ie. x/0) is ready.
```

Comment:
When is this preferred over `wait_for_idle`?

Reply:
it is not meant as a replacement for `wait_for_idle`. this is used to specifically check that unit `/0` is in a ready state, and is to be used in assertions.